app/models/shortener/shortened_url.rb

```diff
@@ -1,148 +1,96 @@
 class Shortener::ShortenedUrl < ActiveRecord::Base
 
+  URL_PROTOCOL_HTTP = "http://"
```

Comment:
is there a reason why we are defaulting to http instead of https?

Reply:
This was the way it was in your last change.  I agree though, probably should be https.  I'll make that update.